packages/grafana-ui/src/components/BigValue/BigValue.mdx

```diff
@@ -0,0 +1,45 @@
+import { Meta, Story, Preview, Props } from '@storybook/addon-docs/blocks';
+import { BigValue } from './BigValue';
+
+<Meta title="MDX|BigValue" component={BigValue} />
+
+# BigValue
+Component for showing a value based on a [DisplayValue](https://grafana.com/docs/grafana/latest/packages_api/data/displayvalue/#displayvalue-interface).
```

Comment:
I suggest adding some info about that optional (if optional? after what we discussed in the weekly about the sparkline being hard-coded I'm not sure :D) sparkline. Also the text mode part of being able to change between showing a name, value or both could be interesting. The text mode select also has "auto" – what exactly does auto mean, do we know?